README.rst

```diff
@@ -1,5 +1,5 @@
-Hypothesis
-==========
+Hypothesis Web Service
```

Comment:
Hmm, I'm not sure about that. This isn't a web service, it's a piece of software that runs a web service.